cdap-ui/app/cdap/styles/common.scss

```diff
@@ -141,6 +141,12 @@ body {
             border: 2px solid $green-03;
             background-color: rgba($green-03, 0.1);
           }
+          > div {
+            padding: 10px;
```

Comment:
After this change, I start seeing the vertical scroll in the profiles list view. Removing this change makes it go away.

![vertical-scroll-profiles](https://user-images.githubusercontent.com/6516002/42117548-0bd9d5d2-7bb2-11e8-8869-6513027b0c88.png)